src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/email/send/EmailSendService.java

```diff
@@ -78,6 +90,7 @@ public void sendPreparedEmail()
             .withRecipientsBCC(bccAddresses)
             .withRecipientsCC(ccAddresses)
             .withContent(message)
+            .withSentDate(new SentDate(sendDate.toLocalDateTime()))
```

Comment:
```suggestion
            .withSentDate(SentDate.ofNow())
```
or something similar

Reply:
unable, see the if-branch above :)

Reply:
Ah no I missed it